pyproject.toml

```diff
@@ -39,11 +42,11 @@ packages = [
 
 [tool.poetry.dependencies]
 djangorestframework = "*"
-inflection = "^0.5.1"
+inflection = "^0.4"
 openapi-spec-validator = "^0.2.9"
-prance = "^0.20.0"
-python = "^3.6.1"
-pyYAML = "^5.3.1"
+prance = "^0.16"
```

Comment:
why the downgrade?

Reply:
Same here. Poetry will install the most recent version within a bound, so we're lowering the lower bound to give users more flexibility.